.circleci/config.yml

```diff
@@ -5,10 +5,15 @@ version: 2
 jobs:
   build:
     docker: # use the docker executor type; machine and macos executors are also supported
-      - image: circleci/golang:1.9
+      - image: circleci/golang:1.10
 
     working_directory: /go/src/github.com/ligato/networkservicemesh
 
     steps:
-      - checkout # check out the code in the project directory
-      - run: echo "hello world" # run the `echo` command
+      - checkout
+      - setup_remote_docker
+      - run: sudo apt-get install yamllint shellcheck python3-pkg-resources
+      - run: go install ./vendor/k8s.io/kube-openapi/cmd/openapi-gen
+      - run: make dep-check
+      - run: yamllint -c .yamllint.yml $(git ls-files '*.yaml' '*.yml' | grep -v 'vendor/')
+      - run: make all && ./scripts/travis-integration-tests.sh packet
```

Comment:
By doing things in this order, you are building the images in the circleci Docker, and then creating the K8S cluster on packet.net. That will then pull the images from Docker Hub and test against those, not the built images from the PR it's supposed to be testing. We hit the same thing with travis-ci and just merged something to transfer the images between the two systems. But in this case, I wonder if it's better to just build the images in the Docker on packet.net? Either way, this needs to be addressed.

Reply:
Currently, travis-ci push every successfully built image to registry... What about to push images always, and pull corresponding image from Docker Hub to use in integration tests?